pkg/api/v1beta1/types.go

```diff
@@ -1079,7 +1079,7 @@ type BoundPods struct {
 	Host string `json:"host" description:"name of a node that these pods were bound to"`
 
 	// Items is the list of all pods bound to a given host.
-	Items []BoundPod `json:"items" description:"list of all pods bound to a given host"`
+	Items []Pod `json:"items" description:"list of all pods bound to a given host"`
```

Comment:
Yes - in fact in #5502 I'm removing BoundPods struct completely.
BTW - I can revert this change if you prefer (this remains from the initial version, but currently I migrated all the usage of BoundPods to PodList).